### PR TITLE
Added my timezone

### PR DESCRIPTION
**Pull request by Darylgolden**
title

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ function App() {
       ["Aathish04", "Asia/Kolkata"],
       ["kilacoda", "Asia/Kolkata"],
       ["naveen521kk","Asia/Kolkata"],
+      ["Darylgolden", "Singapore"]
```
Comment (naveen521kk) [inline src/App.js:28]: is `Singapore` right here? Can you confirm? (see how other's timezone are formatted.)

Comment (Darylgolden) [reply src/App.js:28]: Assuming you're using the [tz database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), yes

Comment (Darylgolden) [reply src/App.js:28]: Oh nvm, it's deprecated

Comment (Darylgolden) [reply src/App.js]: Fixed

**Pull request merged**